app/views/posts/index.html.erb

```diff
@@ -1 +1,43 @@
-<h1>Posts List</h1>
+<div class="user-item">
+  <div class="user-image">
+    <%= image_tag @user.photo, alt: @user.name, width: '100' %>
+  </div>
+  <div class="user-info">
+    <h2><%= @user.name %></h2>
+    <div class="user-posts">
+     <p>Number of Posts: <%= @user.posts_counter %></p>
+     </div>
+  </div>
+</div>
+<div class="posts">
+  <ul>
+    <% @posts.each do |post| %>
+      <li>
+        <div class="post">
+          <h2><%= post.title %></h2>
+          <p><%= post.text %></p>
+          <div class="comment-likes-container">
+            <p>Comments: <%= post.comments.count %></p>
+            <p>Likes: y</p>
+          </div>
+          <% if post.comments.any? %>
+            <ul>
+              <% post.comments.each do |comment| %>
```

Comment:
- [ ] Same here you should use the method post.recent_comments to render 5 most recent comments 👍